### PR TITLE
README: Styleguide for external images

### PR DESCRIPTION
**Pull request by jakob-r**
Should cover all aspects that you have to consider when you include external images. Especially this should be helpfull for all current and future HiWis.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,12 +85,37 @@ Start the code chunk with `block` instead of `r` and add `type='caution'`.
...
+  - `png` files should have reasonable resolution, i.e. the width of a pixel graphic should be between `400px` and `2000px`.
+    If a higher resolution is needed to obtain a readable plot you are probably doing something wrong, e.g. use a pixel graphic where you should use a vector graphic.
+  - Please look at the file size.
+    If your `pdf` or `svg` file is larger than `1MB` it probably contains unnecessary unplotted content or unvectorized parts.
+    If your `png` file is larger than `1MB` the resolution is probably too big.
+
+
+Further aspects:
+* How do I convert `svg` to `pdf`?
+  - Use Inkscape, `rsvg-convert`, `convert` (ImageMagic) or any tool you like.
```
Comment (pat-s) [inline README.md]:
```suggestion
  - Use Inkscape, `rsvg-convert`, `convert` (ImageMagick) or any tool you like.
```

File: README.md [hunk 2]
```diff
@@ -85,12 +85,37 @@ Start the code chunk with `block` instead of `r` and add `type='caution'`.
...
-This way works for the HTML and PDF output.
-In addtion, one can control the width + height of the figure.
-This is not the case for the common markdown syntax `[](<figure>)`.
-
-Always store images also in a vector format (like .svg), even if you do not use them in vector format in the book. Otherwise, we cannot re-use or modify images in the future.
+To include figures follow these rules:
+* Use `knitr::include_graphics()` to add figures instead of markdown syntax `[](<figure>)`. `knitr::include_graphics()` works for the HTML and PDF output and allows to control the width + height of the figure.
+* If available, include the `svg` version in the `Rmd` source, e.g. `knitr::include_graphics("images/some_figure.svg")`.
+* If no `svg` version is available, include the `png` version.
+* Never include a `pdf` figure.
```
[the remark this answers is not in the document]
Comment (jakob-r) [reply README.md]: Maybe I was not clear enough. I am talking about `include_graphics()`. There you should never reference the `PDF`. See a few lines below for "If you add a new figure" it says "For any `svg` file you need to supply a `pdf` version with the exact same name."

[the remark this answers is not in the document]
Comment (pat-s) [reply README.md]: Ah yeah, that was exactly that. Maybe we can clarify the wording here and add an example ala "don't do `include_graphics(image.pdf)`" but only [...]

[the remark this answers is not in the document]
Comment (jakob-r) [reply README.md]: Sollte jetzt klarer sein.

**Pull request merged**